leptonai/cli/objectstore.py

```diff
@@ -0,0 +1,169 @@
+"""
+KV is a module that provides a way to manage kvs on a workspace.
+"""
+
+import click
+import os
+
+from rich.table import Table
+
+from .util import (
+    console,
+    click_group,
+    get_connection_or_die,
+    explain_response,
+    sizeof_fmt,
+)
+from leptonai.api import objectstore as api
+
+
+_max_upload_file_size_limit = 4995 * 1024 * 1024
```

Comment:
out of curiosity, why 4995 such odd number

Reply:
It's 5MB smaller than 5GB as set by R2 https://developers.cloudflare.com/r2/reference/limits/ haha